lib/index.ts

```diff
@@ -4,20 +4,34 @@ import {
   storyblokInit as sbInit,
 } from "@storyblok/js";
 
+import {
+  SbReactSDKOptions,
+  StoryblokBridgeConfigV2,
+  SbReactComponentsMap,
+  StoryblokClient,
+  StoryblokConfig,
+  StoryData,
+} from "./types";
+
 export { default as StoryblokComponent } from "./components/storyblok-component";
 export {
   storyblokEditable,
   apiPlugin,
   useStoryblokBridge,
 } from "@storyblok/js";
 
-export const useStoryblok = (slug, apiOptions = {}, bridgeOptions = {}) => {
-  let [story, setStory] = useState({});
+export const useStoryblok: Function = (
+  slug: string,
+  apiOptions: StoryblokConfig = {},
```

Comment:
The right type should be `StoriesParams`. The `StoryblokConfig` type is to inititalize the js-client